### ISSUE TITLE
Reconsidering the default value for dragFrequency.

### ISSUE THREAD

**Issue by Apkovy** [robinrodricks/vue3-touch-events#41]
Hello, 

I had quite a hard time finding out, why my animation is jerky after migrating from vue2 to vue3.
I am not going into details why, but it was caused mainly by the way the vue3 mounts a mixin, which misled me.

Anyway, please reconsider changing the default value back to vue2 equivalent, whichever it was.

I had to set the `dragFrequency` to `null` or `0` to get the smooth animation again.

I think, that my scenario is pretty common, maybe the prevalent one and this was not happy decision.

Thank you.

**Comment by robinrodricks** [comment]
But firing unlimited events per second is not a good thing.

**Comment by Apkovy** [comment]
Yes, That's why I always use the `requestAnimationFrame` inside the `movingHandler(drag)`.

I don't know what is used in vue2 touch events or vue3 touch events, but this would be at least the plausible solution.

Anyway I would just leave it for the developer to handle this and let the vue3 touch events only dispatch the native events as they are firing in the browser. Maybe puting the warning in the docs would be fine.

Thank you anyway for your attention.

**Comment by robinrodricks** [comment]
Ok accepted. Changed the default to 10 MS (100 times a second).
